auto_py_to_exe/web/js/i18n.js

```diff
@@ -855,5 +921,9 @@ const supportedLanguages = [
         name: 'Turkish (Türkçe)',
         code: 'tr',
     },
+    {
+        name: 'French (Français)',
+        code: 'fr',
+    },

```

Comment:
Can we have this above "Chinese Traditional" please (alphabetical after English)

Reply:
ok